v3/docs/00-style-and-contributor-guidelines/index.mdx

```diff
@@ -0,0 +1,217 @@
+---
+title: Using this guide for writing and style
+slug: /v3/style-guide
+version: 3.0 
+section: docs
+category: style guide
+keywords:
+---
+
+This guide focuses on best practices for writing technical documentation and on the style conventions to use when 
+developing documentation for Parity Technologies products and audiences. 
+The goal of this guide is to help members of the documentation team and any one interested in contributing to documentation 
+write material that is clear, concise, and consistent.
+
+If you can't find the answer to a style, voice, or terminology question in this guide, consult the following resources:
+
+* [Microsoft Style Guide](https://docs.microsoft.com/en-us/style-guide/welcome/)
+* [Chicago Manual of Style](https://www.chicagomanualofstyle.org/book/ed17/frontmatter/toc.html)
+* [Merriam-Webster Dictionary](https://www.merriam-webster.com/)
+
+## General guidance for writing engaging content
+
+There are three keys to writing content that engages the audience:
+
+* Use the _second person_ point of view to directly address the reader.
+* Use an _active voice_ and _present tense_ whenever possible.
+* Use a _conversational tone_ that is not too formal or too chummy.
+
+### Point of view
+
+In most cases, address the reader directly. For tutorials, use either first person plural—we, us, our, ours—or second person point of view. Because tutorials provide a more guided approach to a topic, using the first person plural is a more natural and commonly-accepted practice than in other types of documentation.
```

Comment:
```suggestion
In most cases, address the reader directly. For tutorials, use either first person plural—we, us, our, ours—or second person point of view. 
Because tutorials provide a more guided approach to a topic, using the first person plural is a more natural 
and commonly-accepted practice than in other types of documentation.
```